File: 3228/3228.go
func maximumNumberOfOperationsToMov(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := maximumNumberOfOperationsToMov(root.Left)
	r := maximumNumberOfOperationsToMov(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 3228/3228.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var maximumNumberOfOperationsToMov = function(root) {
    if (!root) return 0;
    return 1 + Math.max(maximumNumberOfOperationsToMov(root.left), maximumNumberOfOperationsToMov(root.right));
};